src/main.go

```diff
@@ -116,6 +119,20 @@ func (r *RadiusCredentials) StartGCWorker() {
 	go r.gcworker()
 }
 
+func (r *RadiusCredentials) accountingCron() *cron.Cron {
+        // hourly accounting cron job
```

Comment:
This is indented with space. Consider adding a save hook that calls `gofmt` (or [`goimports`](https://pkg.go.dev/golang.org/x/tools/cmd/goimports)) for your editor.

Reply:
Fixed.